src/pages/settings/Profile/LoginField.js

```diff
@@ -64,16 +63,6 @@ class LoginField extends Component {
         }
     }
 
-    /**
-    * Bottom margin is not needed for phone/email label when unverified.
-    * When phone/email is not verified, the resend button increases the gap between the label and text,
-    * so the bottom margin is not required.
-    * @returns {Object}
-    */
-    getLabelMargin() {
-        return this.props.login.partnerUserID && !this.props.login.validatedDate ? styles.mb0 : {};
-    }
-
     render() {
         let note;
         if (this.props.type === CONST.LOGIN_TYPE.PHONE) {
```

Comment:
NAB - I feel like we can clean up this whole if-case like this:

```suggestion
        // Check if phone number or email is not validated
        if (this.props.login.partnerUserID && !this.props.login.validatedDate) {
            if (this.props.type === CONST.LOGIN_TYPE.PHONE) {
                note = this.props.translate('loginField.numberHasNotBeenValidated');
            } else {
                note = this.props.translate('loginField.emailHasNotBeenValidated');
            }
        }
        
        
        
        
        
```

Reply:
yeah, I agree that even if I didn't touch this code, I should have made it cleaner. I thought it doesn't make sense to refactor the entire class because we're about to get rid off it.
It is now fixed.